libraries/AP_ADSB/sagetech-sdk/README.md

```diff
@@ -0,0 +1,6 @@
+# Sagetech SDK
+The contents of this folder includes selected functions from the full Sagetech SDK which can be found here:
+https://github.com/Sagetech-Avionics/sagetech-mxs-sdk
```

Comment:
unless you did some modifications, the best approach would be use a submodule to get the sdk

Reply:
We considered this option. There were some small changes to make it work with Ardupilot that were done, and a good number of modules were removed that were unused by the driver as well, but it's probably nothing significant that we couldn't eventually make on our side and go the submodule path. We decided not to go the submodule route mainly because we weren't sure how amenable the Ardupilot community would be to adding new submodules willy nilly. For this pull request though, we request that we simply add this SDK manually like this, and we can make a change to a submodule or other solution in a different pull request.

Reply:
oh gawd no. No more submodules please!!!!